SUPPORTED.md

```diff
@@ -0,0 +1,116 @@
+# Supported devices
+
```

Comment:
We could add the very same text as we have in the README as a preamble, "The following devices have been tested and confirmed as working. If your device is unlisted but working ...".

Reply:
Done